lib/tasks/converter.rake

```diff
@@ -0,0 +1,13 @@
+require_relative "../parsers/convert_to_rspec.rb"
+
+# example usage: bundle exec rake convert:to_rspec\["test/components","spec/views/components"\]
+namespace :convert do
+  desc "task to do some basic conversion of minitest syntax to rspec syntax"
+  task :to_rspec, %i[minitest_folder rspec_folder] => [:environment] do |_, args|
+    message = "minitest_folder and rspec_folder args are required"
```

Comment:
I think the example "usage" comment would be really useful here as it's unclear whether or not the whole path to the test file is required.